src/igvfd/schemas/changelogs/assay_term.md

```diff
@@ -2,6 +2,7 @@
 
 ### Minor changes since schema version 9
 
+* Extend `preferred_assay_title` enum list ot include `scMito-seq`.
```

Comment:
Small typo, the property is plural for Assay Term (singular for Meas Set).

Reply:
fixed